corehq/apps/reports/filters/users.py

```diff
@@ -142,10 +142,12 @@ def user_type_tuple(self, t):
         )
 
     def location_tuple(self, location):
+        location_id = location.location_id
         text = location.get_path_display()
-        if self.display_types:
+        if self.namespace_locations:
```

Comment:
I originally called this `namespace_types`, but using it consistently throughout this file for all types would have some ambiguity - primarily what to do with the static options - and wouldn't be used, so I made it specific to locations.